test/libdnf/rpm/test_solv_query.cpp

```diff
@@ -164,4 +170,15 @@ void RpmSolvQueryTest::test_ifilter_name() {
     /// Test unsupported cmp type
     CPPUNIT_ASSERT_THROW(query8.ifilter_name(libdnf::sack::QueryCmp::GT, names_icontains);,
                          libdnf::rpm::SolvQuery::NotSupportedCmpType);
+
+    /// Test QueryCmp::EQ with two elements
```

Comment:
On more places:
Is it a string for generating documentation?
Comment - `//`
Documentation string - `///`

Reply:
Only comment, I will change it.

Reply:
Fixed